website/src/content/deployment-examples/azure-blob-storage.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Deploying to Azure with Blob Storage
+---
+
+This deployment example refers to [Azure with Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/).
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- An [Azure](https://azure.microsoft.com/free/) account.
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, make sure to provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationid) that you got when you configured the Custom Application in the Merchant Center.
+
+Moreover, you need to provide the [production URL](/api-reference/application-config#envproductionurl) from your Azure project. You can keep the standard Azure Blob Storage Static Website URL `https://<project>.z13.web.core.windows.net/` or provide your custom domain.
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<project>.z13.web.core.windows.net/"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the Application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<project>.z13.web.core.windows.net/"
+    }
+  }
+}
+```
+
+# Connect Azure Blob Storage with GitHub Actions
+
+One of the ways to deploy to Azure is to use [Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/). This service allows hosting a static website, but this needs to be enabled and properly configured in the service. 
+
+Follow the steps in the [Azure Storage Account creator](https://portal.azure.com/#create/Microsoft.StorageAccount) to create a new storage account.
+
+## Configuring static website hosting
+
+Once the storage account is created, go to `Static website` section of the storage account and provide the following settings:
+* Static website: `enabled`
+* Index document name: `index.html`
+* Error document path: `index.html`
+
+## Generating deployment credentials
+
+Follow [the Azure instruction](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-static-site-github-actions?tabs=userlevel#generate-deployment-credentials) to generate deployment credentials. 
```

Comment:
I added that as an external step, since `[Azure Cloud Shell](https://shell.azure.com/)` or Azure CLI installed locally can be used for that, and I didn't want to go into too much details here, but I might if you consider that crucial